syft/codes.py

```diff
@@ -7,6 +7,7 @@ class MSGTYPE(object):
     IS_NONE = 6
     GET_SHAPE = 7
     SEARCH = 8
+    FORCE_OBJ_DEL = 9
```

Comment:
I know it's off-topic for this pull request. What about deriving MSGTYPE from Enum (from enum import Enum). Any reasons for not doing so?

Reply:
I don't think there's a reason not to use an enum... thanks for spotting it.

Reply:
@midokura-silvia I've just tried to change it to an enum and I get serialization/deserialization issues.